wordle-web/layouts/default.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-app>
+    <v-app-bar color="primary">
+      <template v-slot:prepend>
+        <div class="text-h5 ml-5">
+          <v-icon
+            color="secondary"
+            @click="router.push('/index')"

```

Comment:
I think it is a bug because when I click the icon, it sends me to the unfound page. I think you just forgot to remove /index
- Clicking the name and icon of the app take you to the home page ❌

Reply:
Thanks for the catch Sergei! I was debugging something else and added that and forgot to remove it. Fixed it.